README.md

```diff
@@ -1 +1,123 @@
-# Currency-explorer
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-weather-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Weather <a name="about-project"></a>
+
+**Weather App** in this project, i create a weather app using React.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[Search your city]**
+- **[Go to detail page]**
+- **[React Router Dom]**
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS | JS | ES6 | React
+
+- More details about the project can be seen in this [video](https://www.loom.com/share/18778594bc524f93bfeabd8d35a33ca3) 
```

Comment:
- [ ] Please kindly check your video presentation again. I am unable to play it and I cannot get the Transcript as well through which I could follow your presentation. Thanks